app/components/build-header.js

```diff
@@ -42,5 +42,9 @@ export default Ember.Component.extend({
 
   elapsedTime: Ember.computed('item.startedAt', 'item.finishedAt', 'item.duration', function () {
     return durationFrom(this.get('item.startedAt'), this.get('item.finishedAt'));
+  }),
+
+  urlGitHubBranch: Ember.computed('item.repo.slug', 'item.branch.name', function () {
+    return githubBranch(this.get('item.repo.slug'), this.get('item.branch.name'));
   })
```

Comment:
Just wanted to mention I'm a huge fan of that addon and I'd love to see us move in that direction. Decorators are currently stage 2, so it'll still probably be awhile until they're implemented natively, but I think the benefits are high enough for us to justify the investment.